setupenv.sh

```diff
@@ -4,5 +4,5 @@ virtualenv -p python3 env
 echo "unset PYTHONPATH" >> env/bin/activate
 . env/bin/activate
 pip install --upgrade pip
-pip install binpacking flake8 flake8-docstrings mypy prometheus_client pytest pytest-asyncio pytest-cov pytest-mock requests tornado wheel
 pip install git+https://github.com/WIPACrepo/rest-tools
```

Comment:
now that you're installing from requirements.txt, can you drop this line?

Reply:
I think so, as long as rest-tools gets installed first. Otherwise, pip will complain that rest-tools can't be found in PyPI

Reply:
Right, I fixed that by installing it in pip with the '-e' option:
```
pip install -e git+https://github.com/WIPACrepo/rest-tools#egg=rest-tools
```
so when you call `pip freeze` it actually appears correctly.

Reply:
Ahhh, okay, now that I do that, I get that same change in `requirements.txt`:
```
requests-mock==1.5.2
requests-toolbelt==0.9.1
-e git+https://github.com/WIPACrepo/rest-tools@66016820552c6e3452a2d45413772ca53fc4c1e2#egg=rest_tools
setuptools==40.8.0
six==1.12.0
```
I'll make these changes on the /Bundles PR.